xml/hardware_nvdimm.xml

```diff
@@ -285,9 +277,9 @@
     </title> 
     <para>
      In cases where the server will be hosting an application that can directly
-     access large amounts of storage on a byte-by-byte basis, by 
-     <literal>mmap</literal>ing the storage directly into the application's
-     address space.
+     access large amounts of fast storage on a byte-by-byte basis, by 
+     <literal>mmap</literal>ing DAX storage directly into the application's
```

Comment:
Same here: ```<literal>mmap</literal>ing```-  I would avoid using this as a verb, because it might be hard to translate into other languages.

Reply:
Excellent point. I have rephrased the sentence to explain what mmap is and to remove the usage as a verb.